.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rust-analyzer.linkedProjects": ["./Cargo.toml"],
+  "editor.inlayHints.enabled": "offUnlessPressed",
```

Comment:
I understand the cargo features, but why are these two lines needed? I don't think they should solve any specific problems, do they?

Reply:
I quite like the hints, so I think this should be placed in the global vscode config so everyone can choose based on their own preferences, not enforce a purely visual change to everyone in the project.

Reply:
ok.